orchagent/orch.cpp

```diff
@@ -894,12 +894,20 @@ void ZmqOrch::addConsumer(DBConnector *db, string tableName, int pri, ZmqServer
 {
     if (db->getDbId() == APPL_DB)
     {
-        SWSS_LOG_DEBUG("[ZMQ] ZmqConsumer initialize for: %s", tableName.c_str());
-        addExecutor(new ZmqConsumer(new ZmqConsumerStateTable(db, tableName, *zmqServer, gBatchSize, pri), this, tableName));
+        if (zmqServer != nullptr)
```

Comment:
Does this mean, once enabled, it will be ZMQ for all APP_DB tables?

Reply:
No, Once enabled, all orch inherite from ZmqOrch will use ZMQ for APP_DB tables.
Currently only Dash orch inherite from ZmqOrch, so once enabled, all Dash orch will use ZMQ for APP_DB tables.